docs/get-started/labs/create-sealed-secrets.md

```diff
@@ -0,0 +1,76 @@
+---
+slug: create-sealed-secrets
+title: Create Sealed Secrets
+sidebar_label: Create Sealed Secrets
+---
+
+In this lab we will create sealed secrets and see how to securely sensitive information in git repository.
+
+## Creating a Sealed Secret
+
+1. Click on the `Sealed Secrets` in the sidebar.
+
+2. Click on `Create SealedSecret`. This will take you to the page where you can create a sealed secret.
+
+3. Fill in a name for your secret. In this lab we will use the name `secret-credentials`
+
+4. Select the secret type. In this lab we will create a secret of type `opaque`.
+
+5. Add the secret `data`. Add the following key-value pairs:
+
+- `password=helloworld`
+- `username=labs-user`
+
+![Create sealed secret](../../img/create-sealed-secrets.png)
+
+6. Click on `submit`.
+
+7. Click on `Deploy Changes`.
+
+Note that after the secret is created you can not see the values in APL Console anymore:
```

Comment:
The secret value will only be visible at the time of creation or once it has been successfully synchronized with the cluster.